docs/caching.md

```diff
@@ -0,0 +1,26 @@
+# Caching
+
+### Akamai
+
+We use Akamai, a content delivery network, (https://www.akamai.com/), to cache the entirety of consumerfinance.gov (but not our development servers). We invalidate any given page in Wagtail when it is published or unpublished (by hooking up the custom class `AkamaiBackend` (in https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/akamai_backend.py) to Wagtail's frontend cache invalidator (http://docs.wagtail.io/en/v1.9/reference/contrib/frontendcache.html). By default, we clear the Akamai cache any time we deploy. 
```

Comment:
TRIVIAL: To shorten the text we could hide the URLs where applicable. E.g. Have Akamai text as a link with `[Akamai](https://www.akamai.com/)`

Reply:
Does Akamai cover all subdomains of cf.gov? E.g. is beta.consumerfinance.gov behind Akamai? If so we possibly could say `*.consumerfinance.gov`. If not, we could say `www.consumerfinance.gov`.